src/cli/commands/swarm/connect.js

```diff
@@ -18,14 +18,10 @@ module.exports = {
     }
 
     utils.getIPFS((err, ipfs) => {
-      if (err) {
-        throw err
-      }
+      if (err) { throw err }
```

Comment:
Can we please not change the style of those blocks. You were the one who insisted on multiline in the first place, so lets leave it at that please.